pygmt/conftest.py

```diff
@@ -0,0 +1,11 @@
+"""
+conftets.py for pytest.
+"""
+
+import numpy as np
+from packaging.version import Version
+
+# Keep this until we require numpy to be >=2.0
+# Address https://github.com/GenericMappingTools/pygmt/issues/2628.
```

Comment:
What do we do though once we require numpy>2.0 in June 2026?

Reply:
At that time, this file can be removed and we just need to update the doctests to match the numpy 2.0 behavior.